templates/single/assignment/details.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Single attempt page
+ *
+ * @package Tutor\Templates
+ * @author Themeum <support@themeum.com>
+ * @link https://themeum.com
+ * @since 3.8.2
+ */
+
+$description   = get_the_content();
+$has_show_more = strlen( $description ) > 500 ? true : false;
+
+?>
+
+<?php if ( $description ) : ?>
+	<div class="tutor-assignment-description-details tutor-assignment-border-bottom tutor-pb-32 tutor-pb-sm-44">
+		<div id="content-section" class="tutor-pt-40 tutor-pt-sm-60<?php echo $has_show_more ? ' tutor-toggle-more-content tutor-toggle-more-collapsed' : ''; ?>"<?php echo $has_show_more ? ' data-tutor-toggle-more-content data-toggle-height="300" style="height: 300px;"' : ''; ?>>
+			<div class="tutor-fs-6 tutor-fw-medium tutor-color-black">
+				<?php esc_html_e( 'Description', 'tutor' ); ?>
+			</div>
+			<div class="tutor-entry-content tutor-fs-6 tutor-color-secondary tutor-pt-12">
+				<?php echo apply_filters( 'the_content', $description ); //phpcs:ignore ?>
```

Comment:
Instead of this 
`<?php echo apply_filters( 'the_content', $description ); //phpcs:ignore ?>`
Why not use this
`<?php the_content(); ?>`
WordPress caches data so it does not make extra database requests.